tests/unit/test_contracts/capi/action.c

```diff
@@ -13,5 +13,7 @@ void test_action( void ) {
    send_context_free_inline(NULL, 0);
    publication_time();
    current_receiver();
+#if 0
    set_action_return_value(NULL, 0);
```

Comment:
It was a temporary measure because eosio-tester didn't have support for activating protocol features.  I think @tbfleming may have added support since then?

Reply:
It now supports activation via a privileged contract